docs/setup.md

```diff
@@ -106,6 +115,18 @@ $ kubectl get po -n kube-system -l app=local-volume-provisioner
 $ kubectl get pv | grep local-storage
 ```
 
+### Remove a mount point
+
+If we want to remove a mount point, first we need to `umount` the mount point, and then delete the related directories. For example:
+
+```shell
+$ umount /mnt/disks/local-pv01
+$ rm -rf /mnt/disks/local-pv01 
+$ rm -rf /data/local-pv01
+```
```

Comment:
You should also delete the related entries in /etc/fstab at the same time, otherwise the automount may cause problems when the machine restarts.

Reply:
Do we need to add this sentence to the setup.md?

Reply:
Of course, I will add it